streaming/base/dataset.py

```diff
@@ -152,9 +150,10 @@ def __init__(self,
         self.download_timeout = download_timeout
         self.validate_hash = validate_hash or None
 
-        if tdist.is_available() and not tdist.is_initialized() and torch.cuda.is_available() and \
-                'RANK' in os.environ:
-            tdist.init_process_group('nccl')
+        if self.download_retry < 0:
+            raise ValueError('parameter `download_retry` can never be negative.')
```

Comment:
Capitalize

Maybe "Number of download retries must be nonnegative"?

Reply:
Done in https://github.com/mosaicml/streaming/pull/103/commits/bb6649673cbd20eeec770ef30d57668b3ed8afab. I kept the parameter name for the user to know what to change.